### DISCUSSION TITLE
It has been a year already

### DISCUSSION THREAD

**Discussion by raspiduino** [copy/v86#829] [General]
It has been a year since I asked about this  :))

![Screenshot_20230401-112218_Chrome](https://user-images.githubusercontent.com/68118236/229265414-cd0fd024-0522-4cf3-90cb-99d22664c8b1.jpg)

And yeah, on 1/4 I always open this web :))

**Comment by copy** [comment]
Looks like a bug, but should be fixed now.

**Reply by SuperMaxusa** [reply]
"It's not a bug, it's a feature"
